# EC0 - Probabilities

In [16]:
import scipy.stats
import math

beta1 = 4.7
n = 50
p1 = scipy.stats.norm.cdf(-beta1)
pn = 1-math.pow(1-p1, n)
betan = -scipy.stats.norm.ppf(pn)
a1 = scipy.stats.norm.cdf(betan)
an = math.pow(scipy.stats.norm.cdf(beta1), n)
beta1, p1, n, betan, pn, a1, an

(4.7,
 1.3008074539172771e-06,
 50,
 3.826313527780131,
 6.503829991377774e-05,
 0.9999349617000862,
 0.9999349617000862)

In [20]:
# número de horas num ano
hoursyear = 365.25*24
hoursyear, 10*8/hoursyear, 18/365.25

(8766.0, 0.009126169290440338, 0.049281314168377825)

Wind load acts for 10 days during 8 hours in a year = to 80 hours. 
Frequent load combinations act for 0.01 of the time. 
Quasi-permanent load combinations act for 0.5 of the time.

0.02 probability of excedance in one year, corresponds to 0.02*8766=175.32 hours?


In [2]:
3*113

339